### ISSUE TITLE
overview of which cameras are supported

### ISSUE THREAD

**Issue by andybali** [fwestenberg/reolink_dev#543]
is there an overview of which cameras are supported?
I have only found the listing of which are not supported.
Background of my question is the new doorbell, is this supported?
[https://reolink.com/de/product/reolink-video-doorbell/](https://reolink.com/de/product/reolink-video-doorbell/)

**Comment by skynet01** [comment]
It should be supported as it just creates an ONVIF stream which is based on a standard that is followed. The motion sensors might not be supported at first etc. But I dont believe you can actually buy the doorbell as its not released yet.

**Comment by andybali** [comment]
Sorry, my mistake. In the store was the button "add to chart", so I thought you could order. But when you click, it only says "notify me". Then wait and see

**Comment by AsocPro** [comment]
Based on what I’ve been told by support it does sound like it will work basically identical to the rest of their cameras. 

As for the release date I had kept watching it and A week or two ago it had a countdown saying it would be available August 21st. When I went back it showed available for preorder. When I purchased it the estimated ship date was around mid October. I went back to check and it doesn’t show preorder anymore just out of stock. So I would assume that means they sold all the ones that are in their first batch. 
Once mine arrives I can test things to confirm if they work or not.
